src/main/java/xyz/groundx/caver_ext_kas/CaverExtKAS.java

```diff
@@ -103,57 +112,113 @@ public void initKASAPI(String chainId, String accessKeyId, String secretAccessKe
     }
 
     /**
-     * Initialize Node API.
+     * Initialize Node API with Http provider.<br>
      * It sets a url to default endpoint automatically.
      * @param chainId The Klaytn network chain id.
      * @param accessKeyId The access key provided by KAS console.
      * @param secretAccessKey The secret key provided by KAS console.
      */
     public void initNodeAPI(int chainId, String accessKeyId, String secretAccessKey) {
-        initNodeAPI(chainId, accessKeyId, secretAccessKey, URL_NODE_API);
+        initNodeAPI(chainId, accessKeyId, secretAccessKey,false);
     }
 
     /**
-     * Initialize Node API.
+     * Initialize Node API.<br>
+     * It sets a url to default endpoint automatically.<br>
+     * It can choose provider that Http or Websocket via isWebsocketUsed parameter.
+     *
+     * <pre>
+     * {@code
+     * caver.initNodeAPI(1001, "accessKey", "secretAccessKey", false); // use Http provider
+     * caver.initNodeAPI(1001, "accessKey", "secretAccessKey", true); // use Websocket provider
+     * }
+     * </pre>
+     *
+     * @param chainId The Klaytn network chain id.
+     * @param accessKeyId The access key provided by KAS console.
+     * @param secretAccessKey The secret key provided by KAS console.
+     * @param isWebSocketUsed If true, HttpProvider is used. If false, WebsocketProvider is used.
+     */
+    public void initNodeAPI(int chainId, String accessKeyId, String secretAccessKey, boolean isWebSocketUsed) {
+        initNodeAPI(String.valueOf(chainId), accessKeyId, secretAccessKey, URL_NODE_API, isWebSocketUsed);
+    }
+
+    /**
+     * Initialize Node API with Http provider.
      * It sets a url to default endpoint automatically.
      * @param chainId The Klaytn network chain id.
      * @param accessKeyId The access key provided by KAS console.
      * @param secretAccessKey The secret key provided by KAS console.
      */
     public void initNodeAPI(String chainId, String accessKeyId, String secretAccessKey) {
-        initNodeAPI(chainId, accessKeyId, secretAccessKey, URL_NODE_API);
+        initNodeAPI(chainId, accessKeyId, secretAccessKey, false);
+    }
+
+    /**
+     * Initialize Node API.<br>
+     * It sets a url to default endpoint automatically.<br>
+     * It can choose provider that Http or Websocket via isWebsocketUsed parameter.
+     *
+     * <pre>
+     * {@code
+     * caver.initNodeAPI("1001", "accessKey", "secretAccessKey", false); // use Http provider
+     * caver.initNodeAPI("1001", "accessKey", "secretAccessKey", true); // use Websocket provider
```

Comment:
```suggestion
     * caver.initNodeAPI("1001", "accessKey", "secretAccessKey", false); // use Http provider
     * caver.initNodeAPI("1001", "accessKey", "secretAccessKey", true); // use Websocket provider
```
Here also please modify together with parameter name.